### ISSUE TITLE
Calendar 日期初始化持续报警告以及defaultDate设置无效

### ISSUE THREAD

**Issue by wYuJian** [umicro/uView2.0#993]
### 版本
2.0.36

### 转载链接
[www.uviewui.com](https://www.uviewui.com/components/calendar.html)

### 重现步骤
<u-calendar :show="searchDateShow" @confirm="onSearchDateConfirm" @close="searchDateShow=false"
			:defaultDate="defaultDate" :maxDate="maxDate" monthNum="36" :minDate="minDate"></u-calendar>

script：
this.maxDate = uni.$u.timeFormat(Number(new Date()), "yyyy/mm/dd");
this.minDate = (new Date().getFullYear()-3) +"/01/01";
this.defaultDate = [uni.$u.timeFormat(Number(new Date()), "yyyy/mm/dd")];

### 期望的结果是什么？
defaultDate设置有效以及不报警告

### 实际的结果是什么？
报了两千来个警告：new Date("2020-08-16T00:00:00.000Z") 在部分 iOS 下无法正常使用，iOS 只支持 "yyyy/MM/dd"、"yyyy/MM/dd HH:mm:ss"、"yyyy-MM-dd"、"yyyy-MM-ddTHH:mm:ss"、"yyyy-MM-ddTHH:mm:ss+HH:mm" 的格式。以及defaultDate设置无效

**Comment by jinbiaode** [comment]
对 我也遇到了   微信小程序提示：t http://127.0.0.1:24697/appservice/common/vendor.js:2949:16
new Date("2023-11-25T00:00:00.000Z") 在部分 iOS 下无法正常使用，iOS 只支持 "yyyy/MM/dd"、"yyyy/MM/dd HH:mm:ss"、"yyyy-MM-dd"、"yyyy-MM-ddTHH:mm:ss"、"yyyy-MM-ddTHH:mm:ss+HH:mm" 的格式
